src/components/Sections/SectionHeader.tsx

```diff
@@ -57,6 +57,8 @@ export const SectionHeader = ({
       {section && (
         <CozyTheme>
           <ActionsMenu
+            /*
+            // @ts-expect-error ActionsMenu not correctly typed */
```

Comment:
is there a PR in cozy-ui to fix it?

Reply:
I tried to fix every ts error here https://github.com/cozy/cozy-ui/pull/2773 but did not succeed for ActionsMenu and I did not want to spend to much time on it 🙁